### ISSUE TITLE
Can't start light show after setting everything up

### ISSUE THREAD

**Issue by l1chtlaerm** [SaschaK/TuyaMusicSync_Executeable#1]
Issue: I have already set up the Tuya Cloud and gotten Access ID, Access Secret and any Cloud Device ID, which got me the devices as "Online". App just does nothing when pressing Start the light show:

![image](https://github.com/SaschaK/TuyaMusicSync_Executeable/assets/61561129/8ebce061-0258-4572-a7db-c1c4a494240b)

100% positive my lights are on my computer WiFi. Any ideas?

**Comment by SaschaK** [comment]
Hi, please Post a Screenshot of the First tab, too. Please make sure your credentials are Not visible

**Comment by SaschaK** [comment]
I think you forgot to select the Input audio device on the first tab

**Comment by l1chtlaerm** [comment]
![image](https://github.com/SaschaK/TuyaMusicSync_Executeable/assets/61561129/aa7bed76-716a-4be0-a318-8bac613ff0e3)

Positive I'm selecting a sound device. I hit save, reopen just in case, reselect device (should it save? because it isn't saving) and then play the lights show.

**Comment by SaschaK** [comment]
I'm on vacation for the next three weeks. I will check it afterwards

**Comment by herobrine30396** [comment]
Hey, I'm also having this problem. Did you find any way to solve this?

**Comment by l1chtlaerm** [comment]
> Hey, I'm also having this problem. Did you find any way to solve this?

Nope. Couldn't get too much into it so waiting for Sascha to check it

**Comment by SaschaK** [comment]
Hi Guys, sorry for the delay but as far as I mentioned I was on vacation.
Do you hear your music on the speakers? In my case I'm using one of our alexas with an aux cable to play the music in our upper floor and therefore I have to change my sound settings in the windows sound mixer for the mircophone jack: 
![image](https://github.com/SaschaK/TuyaMusicSync_Executeable/assets/4045393/f4b615f8-67a4-4e87-a42b-dff62eed7563)

If you use a service like spofity please take care that no device (excluding your speakers) is connected to your audio jack, because sometimes the soundcard has different output devices for every port. If nothing is works, you can also download the code and Visual Studio Community 2022 to debug the code. 
I tried both options for you:
![image](https://github.com/SaschaK/TuyaMusicSync_Executeable/assets/4045393/09609dcd-d69a-4dd4-a172-19d91396bc4a)

![image](https://github.com/SaschaK/TuyaMusicSync_Executeable/assets/4045393/4cd2ff22-5bcd-44ce-a5d1-169cfc968f44)
![image](https://github.com/SaschaK/TuyaMusicSync_Executeable/assets/4045393/d7583799-2adb-4aab-a956-fceab00ff3c4)

**Comment by herobrine30396** [comment]
Yeah, I can hear the sound on my speakers, and the sound device in the application is set properly. But, when I click start show, it hangs on starting.

**Comment by SaschaK** [comment]
And PC and bulbs are in the same WiFi? If your router provides 2.4GHz and 5GHz WiFi with the same SSID, try to separate them in the router config (for example, I added "_5G" to our 5G SSID or try to configure your PC to use only the 2.4GHz wifi.

**Comment by herobrine30396** [comment]
Both devices are connected to the 2.4G WiFi.
